scripts/test.sh

```diff
@@ -5,7 +5,7 @@
 #   - go vet        (https://golang.org/cmd/vet)
 #   - test coverage (https://blog.golang.org/cover)
 
-COVERALLS_TOKEN=t47LG6BQsfLwb9WxB56hXUezvwpED6D11
+COVERALLS_REPO_TOKEN=t47LG6BQsfLwb9WxB56hXUezvwpED6D11
```

Comment:
This line can be removed. We're already setting `$COVERALLS_REPO_TOKEN` in the Travis CI configurationn.